features/carrinho_compras.features

```diff
@@ -0,0 +1,36 @@
+Feature: carrinho de compras
+    As eu sou um cliente
+    I want escolher os itens que desejo comprar
+    So that eu possa efetuar e receber meu pedido
+
+Scenario: excluir um item do carrinho de compras
+    Given eu estou na "página do carinho de compras"
```

Comment:
This parameter has been repeated along of feature. Create a Background instead it.

Reply:
done